File: Latihan b/java/tim.java
// deklarasi kelas tim
public class tim{
    //dekalrasi variabel lokal
    private String namatim;
    private String asal;
    private int tahun;
    private String pemain;

    // konstruktor
    public tim(){

    }

    //set semua variabel
    public tim(String namatim, String asal, int tahun, String pemain){
        this.namatim = namatim;
        this.asal = asal;
        this.tahun = tahun;
        this.pemain = pemain;
    }

    //get dan set setiap variabel
    public String getnamatim(){
        return namatim;
    }
    public void setnamatim(String namatim){
        this.namatim = namatim;
    }

    public String getasal(){
        return asal;
    }
    public void setasal(String asal){
        this.asal = asal;
    }

    public int gettahun(){
        return tahun;
    }
    public void settahun(int tahun){
        this.tahun = tahun;
    }

    public String getpemain(){
        return pemain;
    }
    public void setpemain(String pemain){
        this.pemain = pemain;
    }

}

File: Latihan b/php/tim.php
<?php
// deklarasi kelas tim
class tim{
	// deklarasi variabel lokal
	private $namatim = "";
	private $asal = "";
	private $tahun = 0;
	private $pemain = "";
	
	// konstruktor
	public function __construct($namatim = "", $asal = "",$tahun = 0, $pemain = ""){
		$this->namatim = $namatim;
		$this->asal = $asal;
		$this->tahun = $tahun;
		$this->pemain = $pemain;
	}

	// get dan set setiap variabel
	public function setnamatim($namatim){
		$this->namatim = $namatim;
	}
	public function getnamatim(){
		return $this->namatim;
	}
	public function setasal($asal){
		$this->asal = $asal;
	}
	public function getasal(){
		return $this->asal;
	}
	public function settahun($tahun){
		$this->tahun = $tahun;
	}
	public function gettahun(){
		return $this->tahun;
	}
	public function setpemain($pemain){
		$this->pemain = $pemain;
	}
	public function getpemain(){
		return $this->pemain;
	}
	
}
?>